security_monkey/alerters/slack_post.py

```diff
@@ -0,0 +1,46 @@
+# Houses functions used to compose slack messages to send to Slack
+#
+# Changes in here will directly affect what messages you receive in slack
+# Written by :: Drew Gallis <drew.gallis@willowtreeapps.com>
+# Date :: June 2018
+#
+
+from slackclient import SlackClient
+import os
+import json
+import requests
+
+
+def postMessage(attachment, type, tech, name):
+
+    # Post to slack (uncomment these lines when you are setting up your token! 
+    # slack_token = os.environ["SLACK_API_TOKEN"] #env variable stored key 
+    
+    slack_token ="thisisatempkey" #use the one above when you are actually doing the setup 
+
+    sc = SlackClient(slack_token)
```

Comment:
This should be moved to a package variable so that it doesn't need to be re-instantiated every time a `postMessage` is called.